test/ttTransactionTestFiller.json

```diff
@@ -0,0 +1,19 @@
+{
+    "VRSTransactionTest" : {
+	"rlp" : "0xf85f80018207d094b94f5374fce5edbc8e2a8697c15331677e6ebf0b0a801ca005062e39bbd54aa760b87cb60f6a1f03099a398a15d943072db37cf7f622806ca0f01e0c8052cb0df363abaf0b24d52b6f62c8aa62bbc5c2e8133a25853fd85f6c",
```

Comment:
the filler file should only consist out of the transaction object.  Because this is a test, the parameters can be invalid or missing. When creating the real test file you just create a rlp out of that and, if valid, a sender.